projects/components/src/lib/togglebutton/toggle-button.component.html

```diff
@@ -0,0 +1,9 @@
+<div
+  class="mas-toggle-button"
+  [ngClass]="{ 'mas-toggle-button--large': size === 'L', 'mas-toggle-button--disabled': disabled}"
+>
+  <div class="mas-toggle-button_switch" (click)="onClick($event, tbutton)">
+    <input #tbutton type="checkbox" class="hidden-accessible" [id]="identifier" [disabled]="disabled" [checked]="checked"/>
```

Comment:
#tbutton should be more specific ex: #toogleButton

Reply:
issue fixed